### DISCUSSION TITLE
UI做的还是蛮棒的

### DISCUSSION THREAD

**Discussion by gousan1013** [share121/inter-knot#606] [其他]
但是不论是回复还是发帖，都需要跳转到github好出戏口牙

**Comment by Bilistar333** [comment]
确实有点麻烦

**Comment by gousan1013** [comment]
而且是因为服务器在github上的缘故嘛，刷新的好慢哦，如果完全使用国内服务器应该会快蛮多（不过到时候就要考虑压力测试和网络攻击的问题力）

**Comment by capacitor1** [comment]
> Reply::而且是因为服务器在github上的缘故嘛，刷新的好慢哦，如果...

雀食，就看站长有没有能力搞个阿里云服务器这种的了 应该是可以的，毕竟绳网对存储和带宽要求不太高。

**Comment by FeiXiao-zm** [comment]
点了，电脑版不知道  手机版来回跳转刷新要好久。。
